src/torcontrol.cpp

```diff
@@ -395,7 +397,7 @@ class TorController
     static void reconnect_cb(evutil_socket_t fd, short what, void *arg);
 };
 
-TorController::TorController(struct event_base* _base, const std::string& _target):
+TorController::TorController(struct event_base* _base, const std::string& _target, const std::string& mode):
```

Comment:
Thought of `enum` in the first place, but then Ocham's razor principle, don't multiply entities etc:) Anyway, done.